demo/common/js/PanoControls.js

```diff
@@ -24,14 +24,15 @@ var PanoControls = (function() {
 		var pieView = new PieView(pieEl);
 		panoViewer.on({
 			"ready": function () {
-				var yawRange = panoViewer.getYawRange();
-				pieView.setState(panoViewer.getYaw(), panoViewer._getHFov(), yawRange[1] - yawRange[0]);
+        var yawRange = panoViewer.getYawRange();
+        var hfov = (180 / Math.PI) * (2 * Math.atan(panoViewer._aspectRatio * Math.tan(panoViewer.getFov() * Math.PI / 360)));
```

Comment:
How about subtracting with a function even though it is the same sentence?

Reply:
Ah... It's a demo. There is no big issue even if you don't change it. But, it's a bit weird to access internal variables in a class.

Reply:
This PR is trying to resolve a previous issue that removing the unnessesary `_getHFov` method breaks the View360 demo.
I won't add additional methods like `getAspectRatio` in this PR as it'll end up increase our minor version.
In other words, I'd like this PR to be just a hotfix. That should be resolved in another PR I think.